content/security/index.adoc

```diff
@@ -40,18 +40,8 @@ Even if you run Jenkins on a private network and trust everyone in your team, se
 == How to Report a Security Vulnerability
 
 If you find a vulnerability in Jenkins, please report it in the issue tracker under the link:https://issues.jenkins.io/browse/SECURITY[SECURITY project].
-This project is configured in such a way that only the reporter, the maintainers, and the Jenkins security team can see the details.
-Restricting access to this potentially sensitive information allows core and plugin maintainers to develop effective security fixes that are safe to apply.
-We provide issue reporting guidelines and an overview of our process on link:reporting[Reporting Security Vulnerabilities].
-
-If you are unable to report using our issue tracker, you can also send your report to the private Jenkins Security Team mailing list:
-`jenkinsci-cert@googlegroups.com`
 
-IMPORTANT: Do not contact the Jenkins security team asking us for compliance documents, certifications, or to fill out a questionnaire.
-We will not respond to such queries.
-If we consider it necessary to provide a statement in response to incidents such as link:/blog/2021/12/10/log4j2-rce-CVE-2021-44228/[log4shell] or link:/blog/2022/03/31/spring-rce-CVE-2022-22965/[SpringShell], you will find a response in our link:/node/[blog].
-
-To show our appreciation for your help, we'll send you link:/security/gift/[a small reward] for privately reported, valid vulnerability reports.
```

Comment:
FTR the redundancy here was deliberate.

Reply:
Would you suggest to keep this redundancy?

Reply:
> Would you suggest to keep this redundancy?

Yes.

> I see the interest of having the information as close as the potential reporters but also the drawback of having two pages talking about the same things in a sense.

One is the quick summary, the other is the full level of detail. (Of course, if we feel we need to add more details to the quick summary, making it too long, like we've kinda started with the `IMPORTANT` block here, its value diminishes.)